ws-nextjs-app/pages/[service]/articles/README.md

```diff
@@ -0,0 +1,50 @@
+# Article Page
+
+## Description
+
+This route is used to render article pages, story pages (STY), photo gallery pages (PG) and media asset pages (MAP).
```

Comment:
Technically the catch-all `[[...]].page.tsx` file in the root of the `pages/[service]` folder renders the STY, PGL, TC2 assets.

The `pages/[service]/articles` route only renders Optimo assets.